wallet/src/repository/wallet/mod.rs

```diff
@@ -15,10 +16,14 @@ mod tests;
 
 use state::State;
 use std::convert::TryFrom;
-use witnet_data_structures::{
-    chain::{Environment, Epoch},
-    transaction::VTTransactionBody,
-};
+use witnet_data_structures::chain::{Environment, Epoch};
+
+/// Internal structure used to gather state mutations while indexing block transactions
+struct TransactionMutation {
```

Comment:
This looks vaguely similar to https://github.com/witnet/witnet-rust/blob/631dc672e62eee8866fa62306f5b8054ed5a6e55/validations/src/validations.rs#L1552-L1556